docs/snapshot-appr-registry.md

```diff
@@ -0,0 +1,152 @@
+# How do I snapshot a Quay Appregistry operator catalog?
+
+## What in tarnation is a _snapshot_?
```

Comment:
Do we want to add another new word here? A `snapshot` is just a registry image / catalog that won't be updated. Maybe we should just use the word `catalog` everywhere?

Reply:
I get your reservation around adding new vocabulary, but I think just saying `catalog` is confusing; i.e. it's hard to tell if we're talking about the thing in appregistry, or the final artifact. I think the word "snapshot" has some implied intent as well.

This won't be around forever anyway. I'd be willing to drop the term if you feel strongly about it.